dashy/src/app/dashboard/widget/platform-status-widget/platform-status-widget.component.css

```diff
@@ -0,0 +1,70 @@
+.widget {
+  position: relative;
```

Comment:
This is the same CSS as the widget above, right? Then you merge them as described in the comment above.